sectionctl.go

```diff
@@ -46,9 +46,10 @@ func bootstrap(c CLI, ctx *kong.Context) {
 
 	colorableWriter := colorable.NewColorableStderr()
 
+	minLogLevel := logutils.LogLevel("INFO") 
```

Comment:
This change doesn't appear to be doing anything to me. The Quiet flag should change our minLogLevel so that we don't have to check the status of c.Quiet when we are logging things.